temoa/components/geography.py

```
@@ -11,7 +11,7 @@
 """
 
 from logging import getLogger
-from typing import TYPE_CHECKING, Iterable
+from typing import TYPE_CHECKING, Any, Iterable
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)